Examples/Modules/gaussian_beam/PICMI_inputs_gaussian_beam.py

```diff
@@ -25,8 +25,10 @@
 grid = picmi.Cartesian3DGrid(number_of_cells = [nx, ny, nz],
                              lower_bound = [xmin, ymin, zmin],
                              upper_bound = [xmax, ymax, zmax],
-                             lower_boundary_conditions = ['periodic', 'periodic', 'open'],
-                             upper_boundary_conditions = ['periodic', 'periodic', 'open'],
+                             lower_boundary_conditions = ['periodic', 'periodic', 'pml'],
```

Comment:
In the current picmi standard, 'open' would be the correct specification. In WarpX, 'open' can be mapped to 'pml'. A separate discussion can be had regarding generalizing the picmi standard to include more BC options (for example adding 'silver-muller').